index.ts

```diff
@@ -415,7 +415,36 @@ app.get("/favorites/:characterId", (req, res) => {
 
 app.get("/blacklist", (req, res) => {
     // e.g. http://localhost:3000/blacklist
-    res.render("blacklist");
+    if (user == null) {
+        return res.status(404).send("User not found");
+    }
+    const blacklist: Blacklist[] = user.blacklist;
+    res.render("blacklist", {blacklist: blacklist});
+})
+
+app.post("/blacklist/:quoteId/delete", async (req, res) => {
+    if (user == null) {
+        return res.status(404).send("User not found");
+    }
+
+    const quoteId = req.params.quoteId;
+    await deleteBlacklist(user, quoteId);
+    await loadUser(user.username);
+
+    res.redirect("/blacklist");
+})
+
+app.post("/blacklist/:quoteId/edit", (req, res) => {
+    if (user == null) {
+        return res.status(404).send("User not found");
+    }
+
+    const quoteId = req.params.quoteId;
+    const newComment = req.body.editComment;
+    editBlacklist(user, quoteId, newComment);
+    loadUser(user.username)
```

Comment:
This also has to be async/await, otherwise the page won't be properly updated upon redirecting it